src/constants/index.js

```diff
@@ -1,13 +1,13 @@
-export const ANNOTATION_DEFAULT_DISK = 'defaults.template.cnv.io/disk';
-export const ANNOTATION_DEFAULT_NETWORK = 'defaults.template.cnv.io/network';
-export const ANNOTATION_FIRST_BOOT = 'cnv.ui.firstBoot';
-export const ANNOTATION_PXE_INTERFACE = 'cnv.ui.pxeInterface';
+export const ANNOTATION_DEFAULT_DISK = 'defaults.template.kubevirt.io/disk';
+export const ANNOTATION_DEFAULT_NETWORK = 'defaults.template.kubevirt.io/network';
+export const ANNOTATION_FIRST_BOOT = 'kubevirt.ui.firstBoot';
```

Comment:
can we rename these so they have the same shape as the rest?

Reply:
done